src/main/java/org/wso2/lsp4intellij/editor/EditorEventManager.java

```diff
@@ -172,10 +172,10 @@
     public Editor editor;
     public LanguageServerWrapper wrapper;
     public List<String> completionTriggers;
-    protected Project project;
-    private RequestManager requestManager;
+    public Project project;
+    public RequestManager requestManager;
+    public TextDocumentIdentifier identifier;
```

Comment:
Same goes here. Keep them as **private** and add **getters**.

Reply:
Added.